SillySCP/API/Features/VolunteerSystem.cs

```diff
@@ -43,18 +43,21 @@ public static void NewVolunteer(RoleTypeId role)
             Volunteers ??= new();
             Volunteers.Add(volunteer);
             Timing.RunCoroutine(ChooseVolunteers(volunteer));
+            
+             var annoucement =                                      // if we add support for other roles this ↓ will need to be changed
```

Comment:
```suggestion
             string annoucement =                                      // if we add support for other roles this ↓ will need to be changed
```

please don't use var, unless v complicated type